CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.16)
 
 include(cmake/GCCToolchain.cmake)
+#include(cmake/ClangToolchain.cmake)
```

Comment:
Maybe remove both lines, if now we directly specify the toolchain file with `--toolchain`.